sdk/metric/internal/sum.go

```diff
@@ -151,3 +165,41 @@ func (s *cumulativeSum[N]) Aggregation() metricdata.Aggregation {
 	}
 	return out
 }
+
+// NewPrecomputedSum returns an Aggregator that summarizes a set of
+// measurements as their pre-computed arithmetic sum. Each sum is scoped by
+// attributes and the aggregation cycle the measurements were made in.
+//
+// The monotonic value is used to communicate the produced Aggregation is
+// monotonic or not. The returned Aggregator does not make any guarantees this
+// value is accurate. It is up to the caller to ensure it.
+//
+// The temporality value is used to communicate the produced Aggregation
+// temporality. The returned Aggregator does not make any guarantees this value
+// is accurate. It is up to the caller to ensure it.
+func NewPrecomputedSum[N int64 | float64](monotonic bool, temporality metricdata.Temporality) Aggregator[N] {
```

Comment:
Is there a reason that temporality is a parameter here, unlike the other Sum Aggregators?

Reply:
no, I'll update.

Reply:
updated.